crypto/rand/rand_vms.c

```diff
@@ -54,7 +54,7 @@ static struct items_data_st {
     {0, 0}
 };
 
-int RAND_poll_ex(RAND_poll_cb rand_add, void *arg)
+size_t RAND_POOL_acquire_entropy(RAND_POOL *pool)
```

Comment:
I don't expect this to compile -- there is still a call to rand_add() at the end of the function's implementation.
Looking at commit 75e2c877650444fb829547bdb58d46eb1297bc1a it may be appropriate to just change it to RAND_add(), but presumably we could bypass that and add it to the pool directly.

Reply:
Converted the code to use the new RAND_POOL_add(), but I can't compile and check it. So I left a TODO in the code:

```C
    /*
     * TODO(DRBG): somebody please check this on vms
     *
     * size of seed is total_length*4 bytes (64bytes). The original assumption
     * was that it contains 4 bits of entropy per byte. This makes a total
     * amount of total_length*16 bits (256bits).
     */
    return RAND_POOL_add(pool,
                         (PTR_T)data_buffer, total_length * 4,
                         total_length * 16);
```